CONTRIBUTING.md

```diff
@@ -0,0 +1,60 @@
+# How to Contribute
+
+Contributing is easy: just change the doc and create a pull request.
+
+The system will automatically initiate some tests on you contribution ("CI").
+Please ensure that these checks succeed.
+
+Prior to merge, a review will be made by a maintainer. If questions appear,
+please be ready to address them.
+
+For a successful contribution, it is important to follow up on the pull
+request.
+
+Authors who provided substantial technical content to a documentation page
+can be mentioned (usually be mentioning themselves) on that page. They
+may include a link to their homepage as well as their email address. The
```

Comment:
I understand this is a project policy decision, but visually we may need a section intended just for this. Maybe a "credits" section that bundles names, contact info, website, etc. At some point we could even come up with a standard format for the info.

Reply:
makes sense - let's work on this later on.

Reply:
Sure. Just wanted to mention it while I was thinking about it.

refs #451 (bread crumb for later review)